activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -100,7 +100,7 @@ def test_indexes_logs_name
     end
 
     def test_table_exists_logs_name
-      @connection.table_exists?('items')
+      ActiveSupport::Deprecation.silence { @connection.table_exists?('items') }
```

Comment:
Maybe you mean "no longer use `table_exists?`" :)
Ditto, I think we should not remove this one. Because after Rails 5.0, `#tables` and `#table_exists?` will be still public API method. After Rails 5.1, behaviors of these methods will be changed but still be public API.